src/service/database/databaseservice.ts

```diff
@@ -214,70 +256,11 @@ export class DatabaseService extends RestService {
 			},
 			HttpMethod.POST);
 			
-			return result;
-		} catch (e){
```

Comment:
where there is no "catch" statement on right part ?

Reply:
What does this mean?